src/dahua.ts

```diff
@@ -125,7 +128,7 @@ class DahuaEvents {
                         const realm = authDetails[0][1].replace(/"/g, '')
                         const nonce = authDetails[2][1].replace(/"/g, '')
                 
-                        const md5 = str => crypto.createHash('md5').update(str).digest('hex')
+                        const md5 = (str: string) => crypto.createHash('md5').update(str).digest('hex')

```

Comment:
I was thinking of doing this in a separate PR but maybe not necessary, I was thinking of separating out the 

```
const md5 = crypto.createHash('md5');
```

so it only creates it once.

Reply:
Ah whoops you can't use the hash after `digest()`, that won't work.

Reply:
I just moved it out to a separate function to make it a bit cleaner.